lapack/gonum/dlanhs.go

```diff
@@ -0,0 +1,66 @@
+// Copyright ©2023 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package gonum
+
+import (
+	"math"
+
+	"gonum.org/v1/gonum/lapack"
+)
+
+// Dlanhs  returns the value of the one norm,  or the Frobenius norm, or
+// the  infinity norm,  or the  element of  largest absolute value  of a
+// Hessenberg matrix A.
+func (impl Implementation) Dlanhs(norm lapack.MatrixNorm, n int, a []float64, lda int, work []float64) (resultNorm float64) {
```

Comment:
I wouldn't name the return, it's clear the float64 must be the norm of A. Also in the Dlanhser interface in the test.